openlibrary/plugins/importapi/tests/test_import_validator.py

```diff
@@ -59,3 +70,20 @@ def test_validate_list_with_an_empty_string(field):
     invalid_values[field] = [""]
     with pytest.raises(ValidationError):
         validator.validate(invalid_values)
+
+
+@pytest.mark.parametrize('field', ['isbn_10', 'lccn'])
+def test_validate_multiple_strong_identifiers(field):
+    """More than one strong identifier should still validate."""
+    multiple_valid_values = valid_values_strong_identifier.copy()
+    multiple_valid_values[field] = ["non-empty"]
+    assert validator.validate(multiple_valid_values) is True
+
+
+@pytest.mark.parametrize('field', ['isbn_13'])
+def test_validate_not_complete_no_strong_identifier(field):
+    """An incomplete record without a strong identifier won't validate."""
```

Comment:
@scottbarnes we need to ensure that a _complete_ record without a strong identifier will still validate, currently that case fails validation:

This is an example test I tried locally  -- Pre-ISBN books with title+author+date should still validate.
```
def test_validate_differentiable_without_strong_identifier():
    """Confirm differentiable pre-ISBN and non LCCN books can still validate."""
    preisbn = {
        "title": "Beowulf",
        "author": {"name": "William Morris"},
        "publishers": ["Kelmscott Press"],
        "publish_date": "1895",
    }
    assert validator.validate(preisbn)

```

For some of those test cases it's hard to tell at a glance what a valid import example  is.

Reply:
completing the testing has moved to #9753